warehouse/upload.go

```diff
@@ -1122,34 +1068,25 @@ func (job *UploadJob) loadTable(tName string) (bool, error) {
 		})
 		return alteredSchema, fmt.Errorf("load table: %w", err)
 	}
+	if loadTableStat.RowsUpdated > 0 {
+		job.statsFactory.NewTaggedStat("dedup_rows", stats.CountType, stats.Tags{
+			"sourceID":       job.warehouse.Source.ID,
+			"sourceType":     job.warehouse.Source.SourceDefinition.Name,
+			"sourceCategory": job.warehouse.Source.SourceDefinition.Category,
+			"destID":         job.warehouse.Destination.ID,
+			"destType":       job.warehouse.Destination.DestinationDefinition.Name,
+			"workspaceId":    job.warehouse.WorkspaceID,
+			"tableName":      tName,
+		}).Count(int(loadTableStat.RowsUpdated))
+	}
 
-	func() {
-		if !generateTableLoadCountVerificationsMetrics {
-			return
-		}
-		var errTotalCount error
-		totalAfterLoad, errTotalCount = job.getTotalCount(tName)
-		if errTotalCount != nil {
-			job.logger.Warnw("total count in table after loading",
-				logfield.SourceID, job.upload.SourceID,
-				logfield.DestinationID, job.upload.DestinationID,
-				logfield.DestinationType, job.upload.DestinationType,
-				logfield.WorkspaceID, job.upload.WorkspaceID,
-				logfield.Error, errTotalCount,
-				logfield.TableName, tName,
-			)
-			return
-		}
-		tableUpload, errEventCount := job.tableUploadsRepo.GetByUploadIDAndTableName(job.ctx, job.upload.ID, tName)
-		if errEventCount != nil {
-			return
-		}
+	tableUpload, errEventCount := job.tableUploadsRepo.GetByUploadIDAndTableName(job.ctx, job.upload.ID, tName)
+	if errEventCount != nil {
+		return alteredSchema, fmt.Errorf("get table upload: %w", errEventCount)
+	}
 
-		// TODO : Perform the comparison here in the codebase
-		job.guageStat(`pre_load_table_rows`, whutils.Tag{Name: "tableName", Value: strings.ToLower(tName)}).Gauge(int(totalBeforeLoad))
```

Comment:
pre_load_table_rows we are not using in any way right now, in Grafana or Alerts.